Source/WTF/wtf/cocoa/RuntimeApplicationChecksCocoa.mm

```diff
@@ -568,66 +564,6 @@ static bool applicationBundleIsEqualTo(const String& bundleIdentifierString)
     return isFeedly;
 }
 
-bool IOSApplication::isPocketCity()
-{
-    static bool isPocketCity = applicationBundleIsEqualTo("com.codebrewgames.pocketcity"_s);
-    return isPocketCity;
-}
-
-bool IOSApplication::isEssentialSkeleton()
-{
-    static bool isEssentialSkeleton = applicationBundleIsEqualTo("com.3d4medical.EssentialSkeleton"_s);
-    return isEssentialSkeleton;
```

Comment:
Unfortunately, this app has not been updated in many years and still requires this quirk. Can you revise the patch to leave this single app in place?